### DISCUSSION TITLE
Running powershell and bash

### DISCUSSION THREAD

**Discussion by RA341** [casey/just#2555] [Q&A] [answered]
I wanna run PowerShell on windows, and bash on Linux, how do I make just 'choose', the appropriate shell on both platforms

**Comment by casey** [comment, marked as answer]
You can use the `windows-shell` setting to change the shell on Windows:

```just
set windows-shell := ["powershell.exe", "-NoLogo", "-Command"]
```

**Reply by RA341** [reply]
Thanks. I am dumb. It would be helpful if this was here: https://github.com/casey/just?tab=readme-ov-file#installation and not buried in the lower sections.
